contrib/python-hypothesis/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "python-hypothesis"
+pkgver = "6.92.2"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = [
+    "python-attrs",
+    "python-pexpect",
+    "python-pytest",
+    "python-pytest-xdist",
+    "python-sortedcontainers",
+]
+depends = ["python-attrs", "python-sortedcontainers"]
+pkgdesc = "Easy to use library for property-based testing"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MPL-2.0"
+url = "https://hypothesis.works/index.html"
+source = f"https://github.com/HypothesisWorks/hypothesis/archive/refs/tags/hypothesis-python-{pkgver}.zip"
+sha256 = "899688a4f7dcca9a6d32ce486a7765db9eef805ee4d84d3333fa6e06276e67b8"
+
+
+def init_build(self):
+    self.make_build_target = "hypothesis-python"
+
+
+def init_check(self):
+    import sys
+    self.env["PYTHONPATH"] = "hypothesis-python/src"
+    self.env["PATH"] = "/usr/bin:" + "f{self.cwd}/testdir/usr/bin"
+    self.do("sh", "-c" ,"python3 -m installer --destdir=test_dir hypothesis-python/dist/*whl")
```

Comment:
the sh -c wrapper isn't needed at all, just call python -m installer itself normally

Reply:
that said it would be nice to make the default pep517 check do the equivalent of a new venv + installing the whl + running the tests, eh..

Reply:
okay, yes would be easier.  Maybe activate venv as well?

Reply:
without the shell it can’t find the whl, even with full path.  Will push